conda_pack/formats.py

```diff
@@ -363,6 +363,10 @@ def zipinfo_from_file(filename, arcname=None):
 class SquashFSArchive(ArchiveBase):
     def __init__(self, fileobj, target_path, arcroot, n_threads, verbose=False,
                  compress_level=4):
+        if shutil.which("mksquashfs") is None:
+            print("Command 'mksquashfs' not found. Please install it, "
+                  "e.g. 'conda install squashfs-tools'.")
+            sys.exit(127)
```

Comment:
This works, but doesn't clean up the `temp_path` created here: https://github.com/conda/conda-pack/blob/4ad334d4338c12bb81b108afbf1116a535e1478b/conda_pack/core.py#L367

Instead raise an Exception, but which a helpful error msg, which will cleanup the path automatically? 

I guess it's pretty rare, so not super important.

Reply:
Thanks for catching. Fixed and tested that tempfiles are really cleaned.